dev/io.openliberty.jakarta.batch.2.1/bnd.bnd

```diff
@@ -0,0 +1,53 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 11
+javac.target: 11
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
+
+Bundle-Name: Java Batch 2.1 API
+Bundle-SymbolicName: io.openliberty.jakarta.batch.2.1; singleton:=true
+Bundle-Description: jakarta.batch 2.1 API (Reference Implementation)
+
+WS-TraceGroup: wsbatch
+
+Private-Package: \
+    io.openliberty.jbatch.jakarta.batch.runtime.internal.resources
+
+Export-Package:\
+ jakarta.batch.api;version="2.1";provide:=true,\
+ jakarta.batch.api.chunk;version="2.1";provide:=true,\
+ jakarta.batch.api.chunk.listener;version="2.1";provide:=true,\
+ jakarta.batch.api.listener;version="2.1";provide:=true,\
+ jakarta.batch.api.partition;version="2.1";provide:=true,\
+ jakarta.batch.operations;version="2.1";provide:=true,\
+ jakarta.batch.runtime;version="2.1";provide:=true,\
+ jakarta.batch.runtime.context;version="2.1";provide:=true
+ 
+Import-Package: org.osgi.util.tracker,\
+	org.osgi.framework,\
+	jakarta.inject;version="2.0",\
+	jakarta.enterprise.util;version="4.0"
+
+# The 2.1.0 batch api bundle has a Bundle-Version of 2.0.0 still in it.
```

Comment:
As of a couple days ago, the v2.1.1 batch-api artifact has been released to [Maven Central](https://search.maven.org/artifact/jakarta.batch/jakarta.batch-api/2.1.1/jar).

@jhanders34 - Not sure whether you'd want to try to update this PR reflecting that, or if you'd rather do this in a later follow-up, in which case I'll open a new issue to remind me or @cgianfra to follow-up.   Thx for catching this Jared !

Reply:
I updated it to 2.1.1